### PR TITLE
Contextual leaderboard - Publication Wed/Thur

### PR DESCRIPTION
**Pull request by clefourrier**
Should be good for a light review :)

### CODE REVIEW

File: leaderboards-on-the-hub-contextual.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Each sample consists of:
+- A text-rich image
+- A human-written instruction (question or imperative task)
+- A human-written reference response 
+
+The dataset is released in two forms: 
+- (a) test dataset with instructions and images, 
+- (b) a validation set of 100 instances from the complete dataset with instructions, reference responses, and images. 
+
+We provide the results on the complete dataset here (and in the paper). The development set would allow the practitioners to iterate on their approaches easily. The evaluation sandbox is present in our github.
```
Comment (NathanHB) [inline leaderboards-on-the-hub-contextual.md:49]: Where are the results, also, the results for what ?

File: leaderboards-on-the-hub-contextual.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+- Proprietary LMMs perform poorly in infographics reasoning after time reading, indicating a gap in their capabilities compared to humans. Notably, GPT-4V, the best performing model, surpasses humans in abstract reasoning, potentially due to exposure to memes and quotes data, but struggles in time-related tasks where humans excel.
+- Open models such as LLaVA-1.5-13B and ShareGPT-4V-7B achieve acceptable ratings only in abstract and natural scenes contexts, suggesting a limitation in their training data diversity.
+- The augmented LLM approach achieves a very low rating of 17.2% which indicates that the dataset instances cannot be solved without precise visual perception.
+
+Our analysis suggests that the practitioners develop enhanced image encoders and create highly accurate image descriptions, to improve the model's perception and mitigate the occurrence of hallucinations. This, in turn, will lead to more effective context-sensitive text-rich visual reasoning.
```
Comment (NathanHB) [inline leaderboards-on-the-hub-contextual.md:78]: I don't really understand that part.

File: leaderboards-on-the-hub-contextual.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+We are taking submissions for both the test set and validation set. Following is the procedure to make submissions for each.
+
+### Validation Set Submission
+
+To submit your validation results to the leaderboard, you can run our auto-evaluation code (Evaluation Pipeline with GPT4), following the instructions [here](https://github.com/rohan598/ConTextual?tab=readme-ov-file#-evaluation-pipeline-gpt-4).
+
+We expect submissions to be json format as shown below:
+
+```json
+{"model_name": {"img_url": "1 or 0 as integer"}}
```
Comment (NathanHB) [inline leaderboards-on-the-hub-contextual.md:99]: Why is it either 0 or 1 for the img_url ?

File: leaderboards-on-the-hub-contextual.md (hunk 1 above)
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:49]: Results are in the leaderboard, I'll make it clearer :)
The results are the model prediction for the instructions + image, like in the example [here](https://con-textual.github.io/static/images/teaser_figure.png)

File: leaderboards-on-the-hub-contextual.md (hunk 3 above)
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:99]: It's 0 or 1 to indicate if the model was correct or wrong on `image_url` iirc

File: leaderboards-on-the-hub-contextual.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+We’d love to evaluate your models too, to help collectively advance the state of vision language models! To submit, please follow our guidelines below.
+
+We hope that this benchmark will help in developing nuanced vision-language alignment techniques and welcome any kind of collaboration! You can contact us here: [Rohan](rwadhawan7@g.ucla.edu) and [Hritik](hbansal@g.ucla.edu), and know more about the team here: [Rohan](https://web.cs.ucla.edu/~rwadhawan7/), [Hritik](https://sites.google.com/view/hbansal), [Kai-Wei Chang](https://web.cs.ucla.edu/~kwchang/), [Nanyun (Violet) Peng](https://vnpeng.net/).
+
+
+
+## How to Submit?
+
+We are taking submissions for both the test set and validation set. Following is the procedure to make submissions for each.
```
Comment (pcuenca) [inline leaderboards-on-the-hub-contextual.md:90]:
```suggestion
We are accepting submissions for both the test and validation sets. Please, follow the corresponding procedure below.
```

File: leaderboards-on-the-hub-contextual.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+### Validation Set Submission
+
+To submit your validation results to the leaderboard, you can run our auto-evaluation code (Evaluation Pipeline with GPT4), following the instructions [here](https://github.com/rohan598/ConTextual?tab=readme-ov-file#-evaluation-pipeline-gpt-4).
```
Comment (pcuenca) [inline leaderboards-on-the-hub-contextual.md:94]:
```suggestion
To submit your validation results to the leaderboard, you can run our auto-evaluation code (Evaluation Pipeline with GPT4), following [these instructions](https://github.com/rohan598/ConTextual?tab=readme-ov-file#-evaluation-pipeline-gpt-4).
```

**Review by pcuenca — APPROVED** [review]

File: leaderboards-on-the-hub-contextual.md (hunk 2 above)
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:76]: @rohan598 if you can clarify :)

File: leaderboards-on-the-hub-contextual.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+- (b) a validation set of 100 instances from the complete dataset with instructions, reference responses, and images. 
+
+We provide the results on the complete dataset here (and in the paper). The development set would allow the practitioners to iterate on their approaches easily. The evaluation sandbox is present in our github.  
+
+## Experiments
+
+For our initial experiments, our benchmark assessed the performance of 13 foundation models (GPT-4V, Gemini-Pro-Vision, LLaVA-1.5). We divided them into three categories: 
+- **Augmented LLM approach**: GPT4 + visual information in the form of OCR of the image and/or dense image captions; 
+- **Closed-Source LMMs**: GPT4V(ision) and Gemini-Vision-Pro; 
+- **Open-Source LMMs**: LLaVA-v1.5-13B, ShareGPT4V-7B, Instruct-Blip-Vicuna-7B, mPlugOwl-v2-7B and so on.
```
[the remark this answers is not in the document]
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:54]: @rohan598 :)

File: leaderboards-on-the-hub-contextual.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+# Introducing ConTextual: How well can your Multimodal model jointly reason over text and image in text-rich scenes?
+
+## Introduction
+
+Models are becoming quite good at understanding text on its own, but what about text in images, which gives important contextual information? For example, navigating a map, or understanding a meme? The ability to reason about the interactions between the text and visual context in images can power many real-world applications, such as AI assistants, or tools to assist the visually impaired. 
+We refer to these tasks as "context-sensitive text-rich visual reasoning tasks".
+
+At the moment, most evaluations of instruction-tuned large multimodal models (LMMs) focus on testing how well models can respond to human instructions posed as questions or imperative sentences (“Count this”, “List that”, etc) over images... but not how well they understand context-sensitive text-rich scenes! 
+That’s why we created ConTextual, a Context-sensitive Text-rich visuaL reasoning dataset for evaluating LMMs. We also released a leaderboard, so that the community can see for themselves which models are the best at this task.
```
[the remark this answers is not in the document]
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:22]: @rohan598

File: leaderboards-on-the-hub-contextual.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+We provide the results on the complete dataset here (and in the paper). The development set would allow the practitioners to iterate on their approaches easily. The evaluation sandbox is present in our github.  
+
+## Experiments
+
+For our initial experiments, our benchmark assessed the performance of 13 foundation models (GPT-4V, Gemini-Pro-Vision, LLaVA-1.5). We divided them into three categories: 
+- **Augmented LLM approach**: GPT4 + visual information in the form of OCR of the image and/or dense image captions; 
+- **Closed-Source LMMs**: GPT4V(ision) and Gemini-Vision-Pro; 
+- **Open-Source LMMs**: LLaVA-v1.5-13B, ShareGPT4V-7B, Instruct-Blip-Vicuna-7B, mPlugOwl-v2-7B and so on.
+
+Our dataset includes a reference response for each instruction, allowing us to test various automatic evaluation methods. For evaluation, we use an LLM-as-a-judge approach, and prompt GPT-4 with the instruction, reference response, and predicted response. The model has to return whether the predicted response is acceptable or not.
```
[the remark this answers is not in the document]
Comment (clefourrier) [reply leaderboards-on-the-hub-contextual.md:56]: Good idea, adding this

File: leaderboards-on-the-hub-contextual.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (rohan598) [reply leaderboards-on-the-hub-contextual.md]: I have added Bliva-Vicuna-7B, Qwen-VL-7B and Idefics-9B also!
No, we did not check Fuyu

File: leaderboards-on-the-hub-contextual.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (rohan598) [reply leaderboards-on-the-hub-contextual.md]: Sure!

File: leaderboards-on-the-hub-contextual.md (hunk 2 above)
Comment (rohan598) [reply leaderboards-on-the-hub-contextual.md]: Thank you for pointing this out! I have rewritten this portion to provide more clarity. In case if it is still unclear, please let me know :)

Open-source models such as LLaVA-1.5-13B and ShareGPT-4V-7B achieve acceptable human ratings only in abstract and natural scene contexts. This indicates that samples (instruction, image pair) belonging to time-reading, infographics, navigation, shopping, web, and mobile usage are out-of-distribution for these models, thereby underscoring a limitation in their training data diversity.

The augmented Large Language Model (LLM) strategy receives a notably low human approval rate of 17.2%, suggesting that the data instances cannot be addressed through visual information converted into text (captions and OCR) alone, necessitating precise visual perception along with fine-grained nuanced vision-language alignment.

Our analysis suggests that the practitioners develop enhanced image encoders, create highly accurate image descriptions, and facilitate fine-grained vision-language alignment to improve the model's perception and mitigate the occurrence of hallucinations. This, in turn, will lead to more effective context-sensitive text-rich visual reasoning.

File: _blog.yml [hunk 9]
```diff
@@ -3575,4 +3575,15 @@
...
-    - LLM 
\ No newline at end of file
+    - LLM 
+
+- local: leaderboard-contextual
+  title: "Introducing ConTextual: How well can your Multimodal model jointly reason over text and image in text-rich scenes?"
+  author: rohan598
+  guest: true
+  thumbnail: /blog/assets/leaderboards-on-the-hub/thumbnail_contextual.png
+  date: Feb 29, 2024
```
Comment (pcuenca) [inline _blog.yml:3585]: Reminder to update before release :)
